addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -28,7 +28,8 @@ WebsiteNewMenu.include({
         return wUtils.prompt({
             id: "editor_new_product",
             window_title: _t("New Product"),
-            input: _t("Name"),
+            input: _t("Product Name"),
+            btn_primary_title: _t("Create"),
```

Comment:
@JKE-be @sbuhl it's already create by default, he just made the button of the default website prompt configurable, so it's continue in regular prompts and create specifically in this one.

Reply:
@sdegueldre Thank you for the precision 👍

Reply:
The button of prompt by default is "Continue"...
Here you override editor_new_product to have Create.

But my idea/question was that all new record modal should be Create and not Continue, so why not change the default value to "Create"...?
![image](https://user-images.githubusercontent.com/4848797/110343676-058de100-802d-11eb-8dee-4f72af316d99.png)


Sorry I don't trust like @sbuhl what I cannot see ;)

Reply:
My bad, I thought it was only for a new product. I am changing the default for every new content to be "Create" :)

Reply:
don't be worry, it was the spec...
But make more sense imo

Reply:
Never trust what you can't see, neither a spec.